### ISSUE TITLE
Aarch64: Duplicated symbols in ARM64Ops.hpp

### ISSUE THREAD

**Issue by knn-k** [eclipse/omr#2761]
I found some more duplicated symbols in `omr/compiler/aarch64/codegen/ARM64Ops.hpp` merged with #2600 as shown below.

`sturb` and `ldurb`:
```
	/* Load/store register (unscaled immediate) */
		sturb,                                                          	/* 0x38000000	STURB     	 */
		ldurb,                                                          	/* 0x38400000	LDURB     	 */
		ldursbx,                                                        	/* 0x38800000	LDURSB    	 */
		ldursbw,                                                        	/* 0x38C00000	LDURSB    	 */
		sturb,                                                          	/* 0x3C000000	STUR      	 */
		ldurb,                                                          	/* 0x3C400000	LDUR      	 */
```

`sturh` and `ldurh`:
```
		sturh,                                                          	/* 0x78000000	STURH     	 */
		ldurh,                                                          	/* 0x78400000	LDURH     	 */
		ldurshx,                                                        	/* 0x78800000	LDURSH    	 */
		ldurshw,                                                        	/* 0x78C00000	LDURSH    	 */
		sturh,                                                          	/* 0x7C000000	STUR      	 */
		ldurh,                                                          	/* 0x7C400000	LDUR      	 */
```

**Comment by knn-k** [comment]
FYI. @aarongraham9 and @jeanlego

**Comment by aarongraham9** [comment]
Thanks for the good find @knn-k! Is this an issue that's blocking you or something that you need imminently for your own PR(s)? Otherwise we'll just correct them in the next Binary Encoding related PR (which work on should start shortly anyway).

**Comment by knn-k** [comment]
@aarongraham9 I am going to make some changes in OpBinary.cpp in #2754, and perhaps it will affect ARM64Ops.hpp, too.  But I don't need this issue to be fixed immediately.

**Comment by knn-k** [comment]
The STURB and LDURB instructions are encoded as `0x38000000` and `0x38400000`, respectively.  I am trying to identify the instructions with the encoding `0x3C000000` (STUR ?) and `0x3C400000` (LDUR ?), but not successful so far.

It is the same for STURH/LDURH.

**Comment by aarongraham9** [comment]
This can be closed via #2779.
